kolibri_dynamic_collections_plugin/assets/src/components/CollectionContentNodeExternalTags.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+  <div class="content-node-external-tags">
+    <KCheckbox
+      v-for="tagId in externalTagOptions"
+      :key="tagId"
+      :checked="tags.indexOf(tagId) >= 0"
+      :label="tagId"
+      @change="onExternalTagCheckboxToggled(tagId, $event)"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  // For now, we hard code a list of external tags that are useful with
+  // Endless Key's Kolibri Explore Plugin.
+  // TODO: Use a tags input with typeahead, similar to
+  //       <https://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/>
+
+  const EXTERNAL_TAGS = [
+    'skills',
```

Comment:
This should be "skill" (not skills) to match what's in the [Explore plugin here](https://github.com/endlessm/kolibri-explore-plugin/blob/e161c47005f1ed00b171a7249f37ffd79b0b9e84/packages/eos-components/src/constants.js#L57) which at the same time was based in the [spec document](https://docs.google.com/document/d/120q-menn7wrEdHBsdfXiMZkHm6MgeFDg8MfDAzIQ0yM/edit) and used in [Heather's spreadsheet](https://docs.google.com/spreadsheets/d/15pbuKaGDxRHSUQJlLpaq436G2Uv4oxr8-HOL-yyJuJg/edit#gid=0) too.

(trying to suggest a change in github but I can't find the button!)